db/seeds.rb

```diff
@@ -6,12 +6,18 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-stache_1 = Stache.create(name: "Handlebar", description: "things", price: 5.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
-stache_2 = Stache.create(name: "Chevron", description: "things", price: 3.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
-stache_3 = Stache.create(name: "Dali", description: "things", price: 10.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
-stache_4 = Stache.create(name: "English", description: "things", price: 9.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
-stache_5 = Stache.create(name: "Fu Manchu", description: "things", price: 4.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
```

Comment:
Let's see if we can set the url equal to a variable to shorten the line. If that doesn't work, we can bitly (that's a verb, right?) the images and get a shorter url.